src/components/LayoutHeader/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { useCallback, useContext } from "react";
+import styled from "styled-components";
+
+import { GameContext } from "../GameProvider";
+import ClassSpellsCarousel from "../CarouselClassSpells";
+import ClassTalentsCarousel from "../CarouselClassTalentsBrowse";
+import TopNavItem from "./TopNavItem";
+import useFlipFlop from "../useFlipFlop";
+
+import { MainScreenList, UniqueCardName } from "../../types";
+import LAYOUT from "../../util/layout";
+
+import trainSpellIcon from "./trainSpellIcon.jpg";
+import talentsIcon from "./talentsIcon.jpg";
+import shopIcon from "./shopIcon.jpg";
+import referenceIcon from "./referenceIcon.jpg";
+
+interface Props {
+  activeList: MainScreenList;
+  setActiveList: (list: MainScreenList) => void;
+}
+
+const Header = ({ activeList, setActiveList }: Props) => {
+  const { addPurchasedCard } = useContext(GameContext);
+  const { value: isSpellbookModalOpen, toggle: toggleSpellbookModal, setOff: hideSpellbookModal } = useFlipFlop();
+  const { value: isTalentModalOpen, toggle: toggleTalentModal, setOff: hideTalentModal } = useFlipFlop();
+
+  const selectSpellbookItem = useCallback(
+    (name: UniqueCardName) => {
+      addPurchasedCard(name);
+      hideSpellbookModal();
+    },
+    [addPurchasedCard, hideSpellbookModal]
+  );
+
+  const toggleListBetweenPowersAndReference = useCallback(
+    () => setActiveList(activeList !== "reference" ? "reference" : "powers"),
+    [activeList, setActiveList]
+  );
+
+  return (
+    <>
+      <NavContainer className="nav">
+        <TopNavItem className="spellbook" iconPath={trainSpellIcon} onClick={toggleSpellbookModal} label="Spells" />
+        <TopNavItem className="talents" iconPath={talentsIcon} onClick={toggleTalentModal} label="Talents" />
+        <TopNavItem className="items" iconPath={shopIcon} onClick={() => console.log("Items Modal")} label="Items" />
+        <TopNavItem
+          active={activeList === "reference"}
+          className="more"
+          iconPath={referenceIcon}
+          onClick={toggleListBetweenPowersAndReference}
+          label="Info"
+        />
+      </NavContainer>
```

Comment:
the point of the text is to disambiguate the functionality, at the moment i dont feel like the text achieves that! I understand going for brevity but we have plenty of room and i think it looks ok.

![image](https://user-images.githubusercontent.com/2404098/169818286-4150ee26-d5de-4a49-9b04-5ac2988e7d91.png)

* Spells -> Train Spells or just "Train"
* Talents -> All Talents
* Items -> Item shop
* Info -> Reference

Reply:
Yeah I like that, will do in follow up PR